.github/workflows/ci-skip.yml

```diff
@@ -74,7 +85,11 @@ jobs:
 
           print('result=verified' if verified else 'result=skipped')
           EOF
-          CHANGED_FILES=$(git diff --name-only ${{ github.base_ref }}...${{ github.event.pull_request.head.sha }} --) python3 ./subclass.py >> $GITHUB_OUTPUT
+          CHANGED_FILES=$(git diff --name-only ${{ github.sha }}...$HEAD_SHA --)
+          echo $CHANGED_FILES
```

Comment:
I added this for debugging purposes; I plan to remove it once we know this workflow is working.